### PR TITLE
Update site-config.xml:

### PR DESCRIPTION
**Pull request by jdrossl**
- Add new ignore-postfix-fields section

https://github.com/craftercms/craftercms/issues/3434

### CODE REVIEW

File: src/main/webapp/repo-bootstrap/global/configuration/samples/sample-site-config.xml [hunk 1]
```diff
@@ -21,11 +21,34 @@
...
+    </form-engine>
+
     <!--
-        Pattern that Studio will use to load extensions from the site repository
-        Required placeholders: ${module}, ${type}, ${name}
+        Pattern that Studio will use to load plugin from the site repository
+        Required placeholders: ${type}, ${name}
      -->
-    <extension-folder-pattern>/config/${module}/extension/${type}/${name}</extension-folder-pattern>
+    <plugin-folder-pattern>/config/studio/plugins/${type}/${name}</plugin-folder-pattern>
```
Comment (sumerjabri) [inline src/main/webapp/repo-bootstrap/global/configuration/samples/sample-site-config.xml:51]: Why this change?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/resources/crafter/studio/upgrade/3.1.x/3.1.2/site/site-config.xslt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                        <xsl:text>force-https</xsl:text>
+                    </xsl:element>
+                    <xsl:element name="field">
+                        <xsl:text>navLabel</xsl:text>
+                    </xsl:element>
+                    <xsl:element name="field">
+                        <xsl:text>expired</xsl:text>
+                    </xsl:element>
+                    <xsl:element name="field">
+                        <xsl:text>key</xsl:text>
```
Comment (avasquez614) [inline src/main/resources/crafter/studio/upgrade/3.1.x/3.1.2/site/site-config.xslt:71]: In which case is `key`, `value` and `items` fields that need input check? I thought those fields where always automatically generated?

**Review by avasquez614 — CHANGES_REQUESTED** [review]

File: src/main/resources/crafter/studio/upgrade/3.1.x/3.1.2/site/site-config.xslt (hunk 2 above)
Comment (jdrossl) [reply src/main/resources/crafter/studio/upgrade/3.1.x/3.1.2/site/site-config.xslt:71]: For example in taxonomies, you can't create the content-type if it doesn't exist

File: src/main/webapp/repo-bootstrap/global/configuration/samples/sample-site-config.xml (hunk 1 above)
Comment (jdrossl) [reply src/main/webapp/repo-bootstrap/global/configuration/samples/sample-site-config.xml:51]: It's something I noticed was wrong, but it's unrelated

**Review by sumerjabri — APPROVED** [review]

**Review by avasquez614 — APPROVED** [review]
